cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -273,7 +274,7 @@ public class ExtractionCheckCommand extends Command {
       description = "Full git commit sha, used for setting a status on a commit in Github.")
   String commitSha;
 
-  @Autowired AuthenticatedRestTemplate restTemplate;
+  @Autowired RestTemplate restTemplate;
```

Comment:
Does the check itself not need the AuthenticatedRestTemplate? i.e. for the request to the Mojito backend to the AI api?